plugins/rt.js

```diff
@@ -70,6 +70,10 @@
  * * `fetch.bnu`: For XHR beacons from fetch API requests, `1` if fetch response body was not used.
  * * `rt.tt`: Sum of load times across session
  * * `rt.obo`: Number of pages in session that did not have a load time
+ * * `ru`: final response URL after any redirects
+ *    - `XMLHttpRequest`: it will be present, even if no redirect happened.
```

Comment:
If there were no redirects, would this always be the same as the request URL? If they're the same, I don't think we'd want to include it on the beacon (twice).